[2024-01-10T18:26:02.849+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: read_to_local_mysql.create_employees_table manual__2024-01-10T18:25:59.743986+00:00 [queued]>
[2024-01-10T18:26:02.878+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: read_to_local_mysql.create_employees_table manual__2024-01-10T18:25:59.743986+00:00 [queued]>
[2024-01-10T18:26:02.879+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-01-10T18:26:02.882+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2024-01-10T18:26:02.883+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-01-10T18:26:02.940+0000] {taskinstance.py:1300} INFO - Executing <Task(MySqlOperator): create_employees_table> on 2024-01-10 18:25:59.743986+00:00
[2024-01-10T18:26:02.952+0000] {standard_task_runner.py:55} INFO - Started process 1899 to run task
[2024-01-10T18:26:02.964+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'read_to_local_mysql', 'create_employees_table', 'manual__2024-01-10T18:25:59.743986+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/slack_mysql_test.py', '--cfg-path', '/tmp/tmpysazhiby']
[2024-01-10T18:26:02.966+0000] {standard_task_runner.py:83} INFO - Job 16: Subtask create_employees_table
[2024-01-10T18:26:03.105+0000] {task_command.py:388} INFO - Running <TaskInstance: read_to_local_mysql.create_employees_table manual__2024-01-10T18:25:59.743986+00:00 [running]> on host 1064147bcaa7
[2024-01-10T18:26:03.252+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=read_to_local_mysql
AIRFLOW_CTX_TASK_ID=create_employees_table
AIRFLOW_CTX_EXECUTION_DATE=2024-01-10T18:25:59.743986+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-10T18:25:59.743986+00:00
[2024-01-10T18:26:03.254+0000] {sql.py:254} INFO - Executing: 
    SELECT jobinfo_test.table_1.company, position, location, review
    FROM jobinfo_test.table_1
    JOIN jobinfo_test.table_2
    ON jobinfo_test.table_1.company = jobinfo_test.table_2.company;
[2024-01-10T18:26:03.275+0000] {base.py:73} INFO - Using connection ID 'mysql_local_test' for task execution.
[2024-01-10T18:26:03.969+0000] {base.py:73} INFO - Using connection ID 'mysql_local_test' for task execution.
[2024-01-10T18:26:04.335+0000] {sql.py:375} INFO - Running statement: 
    SELECT jobinfo_test.table_1.company, position, location, review
    FROM jobinfo_test.table_1
    JOIN jobinfo_test.table_2
    ON jobinfo_test.table_1.company = jobinfo_test.table_2.company;, parameters: None
[2024-01-10T18:26:04.381+0000] {sql.py:384} INFO - Rows affected: 1
[2024-01-10T18:26:04.619+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=read_to_local_mysql, task_id=create_employees_table, execution_date=20240110T182559, start_date=20240110T182602, end_date=20240110T182604
[2024-01-10T18:26:04.741+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-01-10T18:26:04.797+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
